packages/fxa-payments-server/src/components/PaymentConsentCheckbox/index.tsx

```diff
@@ -1,24 +1,22 @@
 import { Localized } from '@fluent/react';
 import LinkExternal from 'fxa-react/components/LinkExternal';
-import { FirstInvoicePreview } from 'fxa-shared/dto/auth/payments/invoice';
 import { urlsFromProductConfig } from 'fxa-shared/subscriptions/configuration/utils';
 import React, { useContext } from 'react';
 
 import AppContext from '../../lib/AppContext';
-import { formatPlanPricing } from '../../lib/formats';
 import { Plan } from '../../store/types';
-import { Checkbox } from '../fields';
+import { Checkbox, OnValidateFunction } from '../fields';
 
 export type PaymentConsentCheckboxProps = {
   plan: Plan;
   onClick?: (event: React.MouseEvent<HTMLInputElement>) => void;
-  invoice?: FirstInvoicePreview;
+  getString?: (id: string) => string;
```

Comment:
Does this need to be a prop? I believe we have l10n functions in `fxa-shared` that can get the translated string. (Also currently I don't think `PaymentConsentCheckbox` consumers are providing a function, so this string won't be translated)

Reply:
Updated.